src/file_writer.rs

```diff
@@ -0,0 +1,63 @@
+use std::fs::File;
+use std::io::{self, Write};
+use std::path::PathBuf;
+
+use crate::common::LineEntry;
+
+// I think that we should create a backup copy
+pub fn write(path: &PathBuf, lines: Vec<LineEntry>) -> io::Result<()> {
```

Comment:
We already have the `fs_utils.rs` file. Please, transfer this method there.

Reply:
OK, and how about backup copy?
Or we can just mention this in the help.

Reply:
We can do it  later.

Reply:
I moved it to `fs_utils`.